README.md

```diff
@@ -12,12 +12,68 @@ A user-friendly IRC bouncer.
 
 ## Usage
 
-    go run ./cmd/sojuctl create-user <username> -admin
+See the man page at `doc/soju.1.scd` for more information.
+
+### Server side
+
+    go run ./cmd/sojuctl create-user <soju username> -admin
     go run ./cmd/soju -listen irc+insecure://127.0.0.1:6667
 
-Then connect with username `<username>/chat.freenode.net` and join `#soju`.
+### Client side
 
-See the man page at `doc/soju.1.scd` for more information.
+soju can operate in two different modes: multi upstream and single 
+upstream.
+
+#### Single connection mode
```

Comment:
s/connection/upstream/

…Or "upstream connection", or "upstream server", or "upstream network", or anything with "upstream" in it :P